src/eterna/pose2D/RNALayout.ts

```diff
@@ -75,6 +75,15 @@ export default class RNALayout {
         return this._root;
     }
 
+
+    public filterForPseudoknots(pairs: number[]): number[] {
```

Comment:
Does this belong here as opposed to being handled somewhere in EPars or something (or just have the function there)?